### PR TITLE
feat: implement react query

### PR DESCRIPTION
**Pull request by smakosh**
## What's new?
- [x] Fix `yarn web gen`
- [x] Install react-query
- [x] Configure react-query
- [x] Configure Graphql-codegen with react-query and get rid of the Apollo plugin
- [x] Switch all hooks to the generated react-query hooks
- [x] Test and make sure everything works fine
- [x] Migrate away from the Subgraph to the new API
- [x] Re-implement hustlers pagination on gangsta party screen
- [x] Remove `useReactiveVar` and `makeVar`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,11 +15,15 @@
...
+		"@types/react-relay": "^11.0.3",
+		"@types/relay-runtime": "^12.0.1",
+		"babel-plugin-relay": "^12.0.0",
 		"lerna": "^3.22.1",
 		"mocha": "^8.4.0",
 		"pinst": "^2.1.6",
 		"ts-node": "^10.0.0",
-		"typescript": "^4.3.2"
+		"typescript": "^4.3.2",
+		"web": "^0.0.2"
```
Comment (tarrencev) [inline package.json:26]: hmm what does this do?

Comment (smakosh) [reply package.json:26]: That was added by you @tarrencev let me remove it

**Comment by tarrencev** [comment]
Couple minor issues:

Dope items should be updated to use `fullname` now.

DOPE items dont show up in `/dope`

image

Opensea listings are now part of the graphql schema, so you can start integrating it. The indexers should be ready soon. For now there won't be any data returned

**Comment by smakosh** [comment]
> 

@tarrencev can you test locally? Because using the Test API works fine and I can see the items when visiting that page and I can see the dopes when visiting `/dope`

**Comment by tarrencev** [comment]
Im having some issues loading the customize page at:

http://localhost:3000/hustlers/0/customize

image

image

**Pull request merged**